README.md

```diff
@@ -0,0 +1,3 @@
+correr en la consola 
+"pip install selenium"
+apretar F5 para ejecutar cada test
```

Comment:
Estas instrucciones están incorrectas; al presionar F5 en la consola, nada sucede.

Reply:
perdon, es F5 en visual

Reply:
Qué Visual? Visual Studio? Si es Visual Studio, qué variante? C#? C++? F#? JSP? ASP? Adicionalmente, qué versión? 2013? 2015? 2017? 2019?
Alternativamente, también podría ser Visual Code...

Reply:
touché, you win this time